core/unit_test/CMakeLists.txt

```diff
@@ -825,6 +825,13 @@ if(Kokkos_ENABLE_CUDA)
       UnitTestMainInit.cpp
       cuda/TestCuda_InterOp_StreamsMultiGPU.cpp
   )
+
+  KOKKOS_ADD_EXECUTABLE_AND_TEST(
+    CoreUnitTest_CudaInterOpGraph
+    SOURCES
+      UnitTestMainInit.cpp
+      cuda/TestCuda_InterOp_Graph.cpp
+  )
```

Comment:
`SYCL` seems to have [`print_graph`](https://github.com/intel/llvm/blob/fbb1fb0010ec128c93148461c951dccf4c01b5b7/sycl/include/sycl/ext/oneapi/experimental/graph.hpp#L317), let's not forget `SYCL`.

Reply:
Done.